src/Vrx_main.cpp

```diff
@@ -228,6 +228,29 @@ void ProcessMSPPacket(mspPacket_t *packet)
     headTrackingEnabled = packet->readByte();
     sendHeadTrackingChangesToVrx = true;
     break;
+#if defined(FUSION_BACKPACK)
```

Comment:
If you add the function stubs as virtuals to the vrxmodule parent class, then we wont need this define